EndlessClient/HUD/Panels/ActiveSpellsPanel.cs

```diff
@@ -559,11 +563,11 @@ protected override void Dispose(bool disposing)
         }
 
         [SupportedOSPlatform("Windows")]
-        private static RegistryKey TryGetCharacterRegistryKey(string accountName, string characterName)
+        private static RegistryKey TryGetCharacterRegistryKey(string accountName, string characterName, string host)
         {
             using RegistryKey currentUser = Registry.CurrentUser;
 
-            var pathSegments = $"Software\\EndlessClient\\{accountName}\\{characterName}\\spells".Split('\\');
+            var pathSegments = $"Software\\EndlessClient\\{host}\\{accountName}\\{characterName}\\spells".Split('\\');
```

Comment:
Registry is only used to load existing item layouts which are overwritten by whatever is in an ini file (if it exists). Registry key path should remain unchanged (for this and for items panel)